MAUI/Chips/Customization.md

```diff
@@ -1180,6 +1222,48 @@ public MainPage()
 
 N> The default value of SelectedChipTextColor is [`Color.White`].
 
+## SelectedChipBackground
```

Comment:
SelectedChipTextColor, Please check the API name

Reply:
API name is SelectedChipTextColor